packages/react-components/react-tooltip/src/Tooltip.content.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { createPortal } from 'react-dom';
+import { motion, Variants } from 'framer-motion';
+import { styled } from '@project44-manifest/react-styles';
+import { ForwardRefComponent } from '@project44-manifest/react-types';
+import { filterDOMProps, mergeProps } from '@project44-manifest/react-utils';
+import { TooltipElement, TooltipProps } from './Tooltip.types';
+
+const StyledMotion = styled(motion.div);
+
+const variants: Variants = {
+	animate: { opacity: 1 },
```

Comment:
Do we think anyone would want finer control over this?

Reply:
More of a curiosity question, why use variants? I see in their docs that these are props on `motion.div`.

Reply:
I was thinking about whether or not we should support overriding the motion props, I dont think it will hurt to expose them. Will update! Will fix the variants too